.storybook/webpack.config.js

```diff
@@ -24,9 +24,9 @@ module.exports = async ({ config }) => {
         options: {
           sassOptions: {
             includePaths: [
-              datePickerIncludePaths,
-              bourbonIncludePaths,
-              neatIncludePaths,
+              ...datePickerIncludePaths,
```

Comment:
Had to make this change to the client template too. Arrays of arrays are not supported.